src/org/omegat/util/gui/ResourcesUtil.java

```diff
@@ -64,11 +66,26 @@ public static Image getImage(final String resourceName) {
     /**
      * Load icon from classpath.
      *
-     * @param iconName
+     * @param imageIcon
      *            icon file name
      * @return icon instance
      */
     public static Image getBundledImage(String imageName) {
         return getImage(RESOURCES + imageName);
     }
+
+    /**
+     * Load default color scheme from classpath.
+     */
+    public static Properties getBundleColorProperties(final String style) {
+        String resourcePath = "/org/omegat/" + "ColorScheme_" + style + ".properties";
+        Properties properties = new Properties();
+        try {
+            InputStream is = ResourcesUtil.class.getResourceAsStream(resourcePath);
```

Comment:
This PR will need some reconciliation with some of the other work you're doing, but I just wanted to point out this for now:

This should use [try-with-resources](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html).